README.md

```diff
@@ -0,0 +1,173 @@
+# Transmogrifier
+The Transmogrifier is a Data Processing Pipeline, which allows for conformation of external data to standardized schemae. It can take data from multiple sources, process it through a series of dataset-specific and broad-level filters, then output the transformed data to any number of sinks (individual or grouped datasets).
+
+The filters can be highly customized to fit any given data standard or requirement and the sinks can be configured to output to a file or upload to a database. 
+
+
+# Purpose
+Transmogrifier is a tool that automates the process of data extraction, transformation, and loading. The Transmogrifier supports extracting data from diverse sources, allows for multiple transformations, and facilitates loading the transformed data to various target destinations.
+
+# Manifest
+The manifest is the JSON file that defines the standardized schemae, data sources (datasets), filters, and sink(s). At the top level, it is an array of objects. Each object contains the following properties:
+- a schema
+- a top-level array of filters
+- a top-level array of sinks (optional)
+- an array of entries
+Within the array of entries, each object contains the following properties:
+- a source
+- an array of filters
+- an array of sinks (optional)
+
+## Features
+The Transmogrifier pipeline is outlined as the following: 
++schema
++filters (broad-level) 
++sinks
++entries
+
+### Filters (broad-level)
+These filters are broad-level filters that will be applied to all of the modified data that comes out of the dataset-specific filters.
+
+The `filters` array defines multiple objects that contain the function needed to transform the data as well as any parameters required. For example, the function "public_art_json_to_json" requires a library parameter to define the location of the library of functions needed to transform the data.
+
+The order that the filters are defined is the order the the data will pass through. For example, in the manifest below, the validator filter is run before the stringify filter.
+
+[Here] (https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/filters/validate.js) is an example of a filter.
+
+### Sinks (broad-level)
+The `sinks` array defines multiple objects that contain the function needed to output the data as well as any parameters required. For example, the function "file_write" requires a path parameter to define the location of the output file.
+
+The broad-level sinks output all of the data into a final destination?
+
+
+### Entries
+An entry is an object that defines the `source`, the `filters` and the `sink` for each particular dataset. 
```

Comment:
think this could be rephrased as "An entry is an object that defines a`source` to read data from, the `filters` to apply to the data, and the `sinks` that define where to write the filtered data"